### PR TITLE
Fixed Zig 0.11 link for stand alone server

### PR DESCRIPTION
**Pull request by JCash**

### CODE REVIEW

File: server/scripts/standalone/setup-standalone-server.sh [hunk 1]
```diff
@@ -40,19 +42,29 @@ fi
 CURL_CMD=/usr/bin/curl
 TMP_DOWNLOAD_DIR=/tmp/_extender_download
 
+function check_url() {
+	local url=$1
+	echo "[check url]" ${url}
+	curl -sSf ${url} > /dev/null
+}
```
Comment (JCash) [inline server/scripts/standalone/setup-standalone-server.sh:49]: Better to do a check of a url, to get a proper error message

File: server/scripts/standalone/setup-standalone-server.sh [hunk 2]
```diff
@@ -108,8 +125,8 @@ PACKAGES=(
 
 ZIG_VERSION=0.11.0
 ZIG_PATH_0_11=${PLATFORMSDK_DIR}/zig-0-11
-ZIG_PACKAGE_NAME=zig-macos-x86_64-${ZIG_VERSION}-dev.3937+78eb3c561.tar.xz
-ZIG_URL=https://ziglang.org/builds
+ZIG_PACKAGE_NAME=zig-macos-x86_64-${ZIG_VERSION}.tar.xz
```
Comment (JCash) [inline server/scripts/standalone/setup-standalone-server.sh:128]: Old archive was no more (403), but it still happily produced an empty folder.
So we didn't notice it until running the server.

File: server/scripts/standalone/setup-standalone-server.sh [hunk 3]
```diff
@@ -40,19 +42,29 @@ fi
...
 function download_package() {
 	local package_name=$1
 	local out_package_name=$package_name
+	local url=${DM_PACKAGES_URL}/${package_name}.tar.gz
 
 	if [[ "$package_name" == *darwin ]]; then
 		out_package_name=$(sed "s/.darwin//g" <<< ${package_name})
 	fi
 
+	# it it has grown old, we want to know as soon as possible
```
Comment (britzl) [inline server/scripts/standalone/setup-standalone-server.sh:60]: "it it" -> "if it"

**Review by britzl — APPROVED** [review]
Looks good. Minor typo in comment.

**Pull request merged**